### PR TITLE
Correction of various internalization issues #1790

### PR DESCRIPTION
**Pull request by mbaillif**
Modification of various panels:
FpagOption,   add S.get
ChronoPanel  add JButton global
OptionsPanel add S.get
WindowsOptions   add JButton global, variables in order of occurrence
AppPreferences, supression unused variable
FpgaBoard  add localChanged
Adapting properties files

### CODE REVIEW

File: src/main/java/com/cburch/logisim/gui/prefs/WindowOptions.java [hunk 1]
```diff
@@ -184,11 +187,11 @@ public WindowOptions(PreferencesFrame window) {
...
-    final var but = new JButton();
-    but.addActionListener(listener);
-    but.setActionCommand(cmdResetWindowLayout);
-    but.setText(S.get("windowToolbarReset"));
-    add(but);
+    ResetWindowLayoutButton = new JButton();
+    ResetWindowLayoutButton.addActionListener(listener);
+    ResetWindowLayoutButton.setActionCommand(cmdResetWindowLayout);
+    ResetWindowLayoutButton.setText(S.get("windowToolbarReset"));
+    add(ResetWindowLayoutButton);
```
Comment (juli-99) [inline src/main/java/com/cburch/logisim/gui/prefs/WindowOptions.java:194]: Please follow the naming convention that variable names should start with a lowercase char.

File: src/main/java/com/cburch/logisim/prefs/FpgaBoards.java [hunk 2]
```diff
@@ -184,6 +184,7 @@ public void actionPerformed(ActionEvent e) {
   private JScrollPane boardPane;
   private JList<String> boardNamesList;
   private JButton addButton;
+  private JLabel ExtBoardPanelTitl;
```
Comment (juli-99) [inline src/main/java/com/cburch/logisim/prefs/FpgaBoards.java:187]: Variables should start with a lowercase letter.

File: src/main/java/com/cburch/logisim/gui/chrono/ChronoPanel.java [hunk 3]
```diff
@@ -59,6 +60,8 @@ public class ChronoPanel extends LogPanel implements Model.Listener {
   private JScrollPane leftScroll;
   private JScrollPane rightScroll;
   private JSplitPane splitPane;
+  private JButton SelButton;
```
Comment (juli-99) [inline src/main/java/com/cburch/logisim/gui/chrono/ChronoPanel.java:63]: Variables should start with a lowercase letter.

**Review by juli-99 — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/logisim/gui/prefs/WindowOptions.java (hunk 1 above)
Comment (mbaillif) [reply src/main/java/com/cburch/logisim/gui/prefs/WindowOptions.java]: Normally, this is corrected

File: src/main/java/com/cburch/logisim/prefs/FpgaBoards.java (hunk 2 above)
Comment (mbaillif) [reply src/main/java/com/cburch/logisim/prefs/FpgaBoards.java]: Normally, this is corrected

File: src/main/java/com/cburch/logisim/gui/chrono/ChronoPanel.java (hunk 3 above)
Comment (mbaillif) [reply src/main/java/com/cburch/logisim/gui/chrono/ChronoPanel.java]: Normally, this is corrected

**Review by BFH-ktt1 — APPROVED** [review]

**Pull request merged**